.github/workflows/main.yml

```diff
@@ -16,35 +16,32 @@ jobs:
     strategy:
       matrix:
         PEXT_BUILD_PORTABLE: [0, 1]
-    name: "AppImage (portable: ${{ matrix.PEXT_BUILD_PORTABLE }})"
-    runs-on: ubuntu-18.04
+        # Operative systems: [ubuntu-18.04, ubuntu-20.04, ubuntu-22.04]
+        # The Ubuntu version 22.04 has problems with Linux script to deploy plugin conda.
+        OPERATIVE_SYSTEM: [ubuntu-latest]
+        # Python versions: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+        # The Python versions 3.6 has problems with QT.
+        # The Python versions 3.10 has problems with Tox Travis.
```

Comment:
With Python 3.6. Using this version cannot get the QT version assigned in the `requirements.txt` (`pyqt5==5.15.7`) because Python / Pip is out of the scope of the last version of QT.

https://github.com/airvzxf/Pext/actions/runs/3315468471/jobs/5476098003

```text
ERROR: Could not find a version that satisfies the requirement pyqt5==5.15.7 (from versions: 5.7.1, 5.8, 5.8.1.1, 5.8.2, 5.9, 5.9.1, 5.9.2, 5.10, 5.10.1, 5.11.2, 5.11.3, 5.12, 5.12.1, 5.12.2, 5.12.3, 5.13.0, 5.13.1, 5.13.2, 5.14.0, 5.14.1, 5.14.2, 5.15.0, 5.15.1, 5.15.2, 5.15.3, 5.15.4, 5.15.5, 5.15.6)
ERROR: No matching distribution found for pyqt5==5.15.7
Error: Process completed with exit code 1.
```

---

For Python 3.10. I researched solutions, but there are no solutions, Tox Travis needs to change a validation about the Python version and import or not the collections. Furthermore, I added an issue in the official repository [Not working with Python 3.10 in Linux](https://github.com/tox-dev/tox-travis/issues/158).

[StackOverflow ➟ Python - can't import Set from sets ("no module named sets")](https://stackoverflow.com/questions/12028204/python-cant-import-set-from-sets-no-module-named-sets)

https://github.com/airvzxf/Pext/actions/runs/3315468471/jobs/5476098553

```text
Traceback (most recent call last):
  File "/home/runner/work/Pext/Pext/.tox/py310/bin/pep257", line 5, in <module>
    from pep257 import main
  File "/home/runner/work/Pext/Pext/.tox/py310/lib/python3.10/site-packages/pep257.py", line 24, in <module>
    from collections import defaultdict, namedtuple, Set
ImportError: cannot import name 'Set' from 'collections' (/opt/hostedtoolcache/Python/3.10.8/x64/lib/python3.10/collections/__init__.py)
ERROR: InvocationError for command /home/runner/work/Pext/Pext/.tox/py310/bin/pep257 pext/ pext_dev/__main__.py (exited with code 1)
___________________________________ summary ____________________________________
ERROR:   py310: commands failed
Error: Process completed with exit code 1.
```

Reply:
Hmm, looking at that error it actually seems to be that pep257 is giving trouble.

I looked pep257 up on PyPI and noticed it hasn't been updated since 2015: https://pypi.org/project/pep257/#history

However, if you go to https://pypi.org/project/pep257/#description and click "[Read the full documentation here](http://pep257.readthedocs.org/)." you see "pydocstyle is a rename and continuation of pep257, a project created by Vladimir Keleshev."

So maybe we can just replace `pep257` with `pydocstyle` in the tox.ini to get a more up-to-date pep257 with Python 3.10 support?

Reply:
I tried to make the change, but it crashes the tests. Could we add a new issue or fix on another branch or pull-request, please? It will help me to have more pull request for the Hacktoberfest, and also is little out of the scope about fix the AppImage and macOS build.

Logs: https://github.com/airvzxf/Pext/actions/runs/3324888315/jobs/5497014678

```text
py39 run-test: commands[3] | pydocstyle pext/ pext_dev/__main__.py
[7134] /home/runner/work/Pext/Pext$ /home/runner/work/Pext/Pext/.tox/py39/bin/pydocstyle pext/ pext_dev/__main__.py
pext/ui_qt5.py:1560 in public method `on_press`:
        D401: First line should be in imperative mood (perhaps 'Execute', not 'Executed')
pext/ui_qt5.py:1575 in public method `on_release`:
        D401: First line should be in imperative mood (perhaps 'Execute', not 'Executed')
ERROR: InvocationError for command /home/runner/work/Pext/Pext/.tox/py39/bin/pydocstyle pext/ pext_dev/__main__.py (exited with code 1)
___________________________________ summary ____________________________________
ERROR:   py39: commands failed
```

Reply:
Ah, `pydocstyle` added new rules that the code should abide by. I guess we can just put `|| true` at the end of the texts and a big `FIXME: Fix tests` above it, at least for now yeah

Reply:
Oh, I see now you already went back to pep257. That works too for now then.